packages/ember-cli-eyeglass/src/index.ts

```diff
@@ -77,19 +77,20 @@ function isUsingEmbroider(app: any): boolean {
   return !!(dependencies["@embroider/core"] || devDependencies["@embroider/core"]);
 }
 
-function embroiderEnabled(): boolean {
-  return g.EYEGLASS.projectInfo.usingEmbroider;
+//eslint-disable-next-line @typescript-eslint/no-explicit-any
+function embroiderEnabled(config: any): boolean {
+  return config.embroiderEnabled ?? g.EYEGLASS.projectInfo.usingEmbroider;
```

Comment:
If we go down the route this PR recommends, should we have both: 'isUsingEmbroider' dependency check, and this flag?

Reply:
The old flag being renamed to `hasEmbroiderDependencies` could help with the above mentioned ambiguity.

Reply:
I'd be worried about dropping the current behavior since that would be backwards-incompatible. If there's a way we can do this detection in a more reliable way, I'd prefer that over asking the app to declare it manually, since then apps wouldn't need to make any changes.

I added some info on this behavior in the readme, let me know if the description is sufficient. 👍

Reply:
Renamed `isUsingEmbroider` to `hasEmbroiderDependencies`. (Sorry, didn't show me your second comment until I did a hard refresh. 🤦 )